manifests/init.pp

```diff
@@ -324,6 +324,10 @@
 
   $template_snmpd_conf          = 'snmp/snmpd.conf.erb'
   $template_snmpd_sysconfig     = "snmp/snmpd.sysconfig-${facts['os']['family']}.erb"
+  $template_snmpd_service = $facts['os']['release']['full']  ? {
```

Comment:
I don't see the distinction between the two here

Reply:
Ah, now I see the extension differs. I'd make it more explicit it's a drop in. It also doesn't appear to be used elsewhere so:
```suggestion
  $template_snmpd_service_dropin = "snmp/snmpd.service-dropin-${facts['os']['family']}.epp"
```